jyulb/fileio/isothermal/tests/test_jyu_engine.py

```diff
@@ -46,6 +48,70 @@ def cleanup(self):
         os.chdir(self.saved_path)
         shutil.rmtree(self.temp_dir)
 
+    def engine_factory(self):
+        return lb.JYUEngine()
+
+    def create_dataset_items(self):
+        """ Not applicable to JYU-LB
+        """
+        pass
+
+    def check_instance_of_dataset(self, ds):
+        self.assertIsInstance(ds, ABCLattice,
+                              "Error: Dataset must be ABCLattice!")
+
+    def test_delete_dataset(self):
+        """ JYU-LB does not support adding multiple datasets, therefore
+        this test is overriden
+        """
+        engine = self.engine_factory()
+        engine.add_dataset(self.create_dataset("test"))
+        for ds in engine.iter_datasets():
+            engine.remove_dataset(ds.name)
+            with self.assertRaises(ValueError):
+                engine.get_dataset("test")
```

Comment:
I see you fixed this line from using `ds.name`.  We should do this fix in simphony-common as well.